### PR TITLE
Add eduGAIN registration content to README

### PR DESCRIPTION
**Pull request by bradleybeddoes**
Requesting review by all of @ausaccessfed/developers simply so you know how to do this if I am not around.

### CODE REVIEW

**Review by rianniello — APPROVED** [review]

**Comment by mikelkew** [comment]
LGTM, extra documentation definitely helpful. Curious to know if there's any specific reason this can't be simplified with 2 rake tasks though?

**Review by chrisjones120 — APPROVED** [review]

**Review by rcaught — CHANGES_REQUESTED** [review]
Can we use `!` bang methods on all `create` and `save` calls?

**Comment by bradleybeddoes** [comment]
> Can we use `!` bang methods on all `create` and `save` calls?

Your command is my wish sir @rcaught - https://github.com/ausaccessfed/saml-service/pull/234/commits/b5da992c43c1d3c0c70239036df4dffc43ecc751

**Comment by bradleybeddoes** [comment]
> Curious to know if there's any specific reason this can't be simplified with 2 rake tasks though?

A toxic combination of "we've always done it this way" and laziness.

Let me have a poke around, should be doable.

**Comment by bradleybeddoes** [comment]
To make sir @mikelkew happy I have provided rake tasks for this now, he'll love them. 

README updated.

Have validated all logic against test federation SAML service and ensured flows through to metadata. Unit tests will have to be a thing for another day though.

**Review by rcaught — CHANGES_REQUESTED** [review]
If I say the rake tasks are missing bangs will you hate me?

**Comment by mikelkew** [comment]
> To make sir @mikelkew happy I have provided rake tasks for this now, he'll love them.

Nice one, that I do!

Noting also that `raise_on_save_failure` is true be default for `create`/`save` in Sequel @rcaught.

**Comment by rcaught** [comment]
> > To make sir @mikelkew happy I have provided rake tasks for this now, he'll love them.
> 
> Nice one, that I do!
> 
> Noting also that `raise_on_save_failure` is true by default for `create`/`save` in Sequel @rcaught.

Oh right!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+#### Publishing an AAF registered SP
+
+To export an AAF registered SP to eduGAIN:
+
+1. Get a session on `<SAML Service Host>` 
+1. Become the `saml` user, `$> sudo -iu saml`
+1. Source `.app-env`
+1. Change to `repository`
+1. Run `rake edugain:publish_sp\['https://example.edu.au/shibboleth','https://example.edu.au/information_url'\]` replacing `https://example.edu/shibboleth` with the target EntityID
```
Comment (mikelkew) [inline README.md:38]: Escaping the square brackets doesn't seem to be necessary for me in `zsh`, `bash` or `sh`, but I guess no harm to use them regardless.

**Comment by bradleybeddoes** [comment]
> Noting also that `raise_on_save_failure` is [true by default](https://sequel.jeremyevans.net/rdoc/classes/Sequel/Model/ClassMethods.html#attribute-i-raise_on_save_failure)

Yep, relying on this for the most part but then being explicit in the final `save` to be 100% sure.

**Review by mikelkew — APPROVED** [review]

**Review by rianniello — APPROVED** [review]

**Review by rcaught — APPROVED** [review]

**Pull request merged**